src/routes/__error.svelte

```diff
@@ -8,34 +8,52 @@
 </svelte:head>
 
 <section>
-  <img src="/assets/png/404-h.png" class="wide" alt="404 - Page Not Found" />
-  <img src="/assets/png/404-v.png" class="mobile" alt="404 - Page Not Found" />
+  <em>404</em>
+  <h1>Frank can't find where you're going!</h1>
+  <img src="/assets/png/lost-frank.png" class="wide" alt="404 - Page Not Found" />
+  <img src="/assets/png/lost-frank.png" class="mobile" alt="404 - Page Not Found" />
 </section>
 
 <style>
   section {
     min-height: 100vh;
     display: flex;
+    flex-direction: column;
     justify-content: center;
     align-items: center;
     margin: 0 24px;
+    color: var(--acm-light);
   }
-
-  section .mobile {
-    display: none;
+  section h1 {
+    margin-bottom: 2em;
+    align-items: center;
+    text-align: center;
+  }
+  section em {
+    align-items: center;
+    text-align: center;
+    font-size: 50px;
+    margin-bottom: 0.5em;
+    margin-top: 2em;
   }
```

Comment:
To make the `<em>` stand out like it does in the original 404 graphic, we could add a [text-shadow property](https://developer.mozilla.org/en-US/docs/Web/CSS/text-shadow) to this CSS block: 

Mode | Dark | Light
---|---|---
Before text-shadow | ![image](https://user-images.githubusercontent.com/31261035/169717052-76d27ac9-479a-4059-a10d-5e8e260d3807.png) | ![image](https://user-images.githubusercontent.com/31261035/169717063-49512868-a0e7-4897-9cff-ec8d2b43d2ab.png)
After text-shadow | ![image](https://user-images.githubusercontent.com/31261035/169716995-5d60ce45-d94b-4e4f-b7de-3c90720fb6cb.png) | ![image](https://user-images.githubusercontent.com/31261035/169716969-a33cc18a-abcd-4253-9bf2-64fbd92ea027.png)

```suggestion
    text-shadow: 2px 2px var(--acm-blue);
  }
```

Reply:
I used `--acm-blue` for this suggestion, but the original 404 graphic used `#92c4df`.

![image](https://user-images.githubusercontent.com/31261035/169718924-13188ca7-6849-4636-9e39-d3835c278afa.png)

Reply:
> I used `--acm-blue` for this suggestion, but the original 404 graphic used `#92c4df`.
> 
> ![image](https://user-images.githubusercontent.com/31261035/169718924-13188ca7-6849-4636-9e39-d3835c278afa.png)

To better match the original 404 graphic, you are able to add some more weight, lighten the color, and increase the font size as needed (while keeping in mind how the element responds on mobile and wide screens).
```suggestion
    /* #92c4df is slightly lighter than var(--acm-blue) */
    /* see <https://etok.codes/acmcsuf.com/pull/458/files#r878929505> */
    text-shadow: 2px 2px #92c4df;
    font-weight: 600;
  }
```

Reply:
Done